src/pages/card-view/card-view-cards.tsx

```diff
@@ -30,6 +38,111 @@ function CardViewSection(props: Props) {
   );
 }
 
+function ChangeIDBar(props: { currentCard: CardWithRelations }) {
+  const { currentCard } = props;
+
+  /* These are required to be able to filter a card list.*/
+  const metadata = useStore(selectMetadata);
+  const lookupTables = useStore(selectLookupTables);
+
+  const allCards = Object.values(metadata.cards); /* a list of all the cards. */
+
+  var searchCode = currentCard.card.pack_code;
+
+  /* Confirming that the pack code is accurate to the new format. */
+
+  const cyclesWithPacks = useStore(selectCyclesAndPacks);
+  const currentPack = cyclesWithPacks.find(
+    (pack) =>
+      pack.code === metadata.packs[currentCard.card.pack_code].cycle_code,
+  );
+
+  if (currentPack?.reprintPacks) {
+    const targetType = currentCard.card.encounter_code ? "encounter" : "player";
+    const reprint = currentPack.reprintPacks.find(
+      (pack) => pack.reprint?.type === targetType,
+    );
+
+    if (reprint) {
+      searchCode = reprint.code;
+    }
+  }
+
+  /* A filtered list of all the cards, specifically, all the cards from the same set as the current card, sorted by their "Position" (set code), b sides removed.*/
+  const filteredCards = allCards
+    .filter((card) =>
+      filterPackCode([searchCode], metadata, lookupTables)?.(card),
+    )
+    .filter((card) => filterBacksides(card));
+  filteredCards.sort(sortByPosition);
+
+  /* The list index of the current card in the list. Using this, I can find the cards before and after the current card, even if there's a jump in set position. */
+  const cardListIndex = filteredCards.findIndex(
+    (card) => card.code === currentCard.card.code,
+  );
+
+  return (
+    <div style={{ display: "flex", justifyContent: "space-between" }}>
+      <div style={{ width: "200px" }}>
```

Comment:
Why do we need static widths here? This seems arbitrary and does not work well on mobile.